libs/movex/package-lock.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file got generated when you ran 'npm install' I believe. We use `yarn` for the whole project so we don't need this, as it'll create conflicts. Could you please remove it? 🙏